scrapers/JeffsModels.yml

```diff
@@ -0,0 +1,30 @@
+name: "JeffsModels"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - jeffsmodels.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="updateInfo"]/h2/text()
+      Details:
+        selector: //div[@class="updateDescription"]/p/text()
+      Performers:
+        Name: //div[@class="updateModels"]/a/text()
+      Image:
+        selector: //a[@class="hi iconPlay"]/img[@alt="Main Sample"]/@src
+        postProcess:
+          - replace:
+             - regex: ^
+               with: "https://jeffsmodels.com"
+      Studio:
+        Name:
+          fixed: "Jeff's Models"
+      Date:
+        selector: //div[@class="updateMeta"]/span[@class="updateMetaDate"]/span[@class="updateDate"]/text()
```

Comment:
For the date there seems to be only one span with that class
Is there a reason not to use the simpler `//span[@class="updateDate"]/text()` ?

Reply:
This makes sense. Thanks for finding that.